src/geo/leaflet/leaflet-cartodb-webgl-layer-group-view.js

```diff
@@ -0,0 +1,49 @@
+var TC = require('tangram.cartodb');
+var LeafletLayerView = require('./leaflet-layer-view');
+
+var LeafletCartoDBVectorLayerGroupView = window.L.TileLayer.extend({
+  includes: [
+    LeafletLayerView.prototype
+  ],
+
+  options: {
+    minZoom: 0,
+    maxZoom: 28,
+    tileSize: 256,
+    zoomOffset: 0,
+    tileBuffer: 50
+  },
+
+  events: {
+    featureOver: null,
+    featureOut: null,
+    featureClick: null
+  },
+
+  initialize: function (layerGroupModel, map) {
+    LeafletLayerView.call(this, layerGroupModel, this, map);
+    layerGroupModel.bind('change:urls', this._onURLsChanged, this);
+
+    this.tangram = new TC(map);
+
+    layerGroupModel.each(this._onLayerAdded, this);
+    layerGroupModel.onLayerAdded(this._onLayerAdded.bind(this));
+  },
+
+  onAdd: function (map) {
+
+  },
+
+  _onLayerAdded: function (layer) {
+    var self = this;
+    layer.bind('change:meta', function (e) {
+      self.tangram.addLayer(e.attributes);
+    });
+  },
+
+  _onURLsChanged: function (e, res) {
+    this.tangram.addDataSource(res.tiles[0]);
```

Comment:
I agree with you, but actually it is only happens one time per layer because this only works on the embed map, to make a full implementation with more use cases (ex. use it in Builder) it will be necessary.